File: app/src/main/java/com/br/apigithub/beans/Owner.java
package com.br.apigithub.beans;

import com.google.gson.annotations.SerializedName;

/**
 * Created by rlima on 07/10/18.
 */

public class Owner {
    @SerializedName("id")
    private Integer id;
    @SerializedName("login")
    private String login;
    @SerializedName("avatar_url")
    private String avatarUrl;

    public Owner() {
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public void setAvatarUrl(String avatarUrl) {
        this.avatarUrl = avatarUrl;
    }
}

File: app/src/main/java/com/desafioandroid/getirepos/data/dto/Owner.kt
package com.desafioandroid.getirepos.data.dto

import com.google.gson.annotations.SerializedName

data class Owner(
    @SerializedName("login")
    val login: String,
    @SerializedName("id")
    val id : Long,
    @SerializedName("node_id")
    val nodeId: String,
    @SerializedName("avatar_url")
    val avatarUrl: String,
    @SerializedName("gravatar_url")
    val gravatarUrl: String
)
